pulpcore/pulpcore/app/settings.py

```diff
@@ -48,8 +48,6 @@
 # Application definition
 
 INSTALLED_APPS = [
-    # Dynamic configuration with Dynaconf
-    'dynaconf.contrib.django_dynaconf',
```

Comment:
It might be nice commented out rather than remove because this is the logical place to see which apps are being used.

Reply:
I did that and added a link to the dyanconf issue everywhere. Thanks for the suggestion.